local/bin/py/build/configurations/pull_config.yaml

```
@@ -1130,6 +1130,18 @@
       options:
         dest_path: '/security/default_rules/'
 
+  - repo_name: infrastructure-resources
+    contents:
+    - action: pull-and-push-folder
+      branch: prod
+      globs:
+        - "generated/md/external/*.md"
+      options:
+        dest_dir: '/security/cspm/custom_rules/'
+        path_to_remove: 'generated/md/external/'
+        front_matters:
+          dependencies: ["https://github.com/DataDog/infrastructure-resources/blob/prod/README.md"]
```

because infrastructure-resources is an internal repo, we don't want to enable the Edit button on these pages. Websites suggested
```suggestion
        front_matters:
          dependencies: []
```